test/metabase/api/search_test.clj

```diff
@@ -547,10 +559,10 @@
 
 (deftest table-test
   (testing "You should see Tables in the search results!\n"
-    (mt/with-temp Table [_ {:name "Round Table"}]
+    (mt/with-temp Table [_ {:name "RoundTable"}]
```

Comment:
I had other search results that matched `Table` that were polluting the results here. But I don't have anything named `RoundTable` :upside_down_face: